journal/202112.md

```diff
@@ -0,0 +1,477 @@
+- [Media](#media)
+- [Discussions](#discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+
+## v1.7 Release Candidates
+
+{ quick info how to join testing, testnet coins, how to report, etc }
+
+
+## Vote on treasury spends
+
+{ maybe have a standalone and more visible notice to prompt more votes on tspends https://twitterc.com/decredproject/status/1472050886830702597 }
+
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+_dcrd is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+Development work for the upcoming "Change PoW/PoS Subsidy Split To 10/80" consensus change has been completed, including the agenda [definitions](https://github.com/decred/dcrd/pull/2847), the [consensus vote](https://github.com/decred/dcrd/pull/2848) with new consensus rules and tests, and the [DCP-10](https://github.com/decred/dcps/pull/23) document with technical details. A small geek bonus of this change is it will increase the theoretical supply limit from the uncomfortable 20,999,999.99800912 to a more satisfying 21,000,000.00015952 DCR.
+
+The new `primitives` package has been kickstarted with initial [proof-of-work](https://github.com/decred/dcrd/pull/2788) functions, calculations of [merkle tree](https://github.com/decred/dcrd/pull/2826) roots, and functions for generating and verifying [inclusion](https://github.com/decred/dcrd/pull/2827) proofs (along with full tests, benchmarks, and docs). New PoW functions build upon the new optimized `uint256` package and are much faster than the ones in use currently. `primitives` is a large refactoring [plan](https://github.com/decred/dcrd/issues/2786) to collect core consensus code in one place.
+
+Other merged changes:
+
+- introduced [`AssumeValid`](https://github.com/decred/dcrd/pull/2839) parameter and config option to skip certain validation checks prior to the specified block (which is assumed valid). Compared to using checkpoints for same purpose, `AssumeValid` allows to skip more validation work and gives greater flexibility to node operators and developers (for testing).
+- added an additional child key [derivation](https://github.com/decred/dcrd/pull/2845) method that strictly conforms to the [BIP-32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) standard, to be used by DCRDEX. Decred's current method differs from the standard for legacy reasons.
+- fixed logic for preventing unnecessary [reorgs](https://github.com/decred/dcrd/pull/2840)
+- fixed a bug in CPU [mining](https://github.com/decred/dcrd/pull/2838) (used in simnet testing)
```

Comment:
I also don't think this is useful information for general consumption.  It only applies to very small number of developers who are already well aware of it.

Reply:
Removed.

This feedback helps to achieve better balance between listing what is notable enough and saving reader's time, since I don't always guess notability. Thanks.